openslides/__main__.py

```diff
@@ -129,6 +129,10 @@ def get_parser():
         '--use-geiss',
         action='store_true',
         help='Use Geiss instead of Daphne as ASGI protocol server.')
+    subcommand_start.add_argument(
```

Comment:
I do not like this extra command line option. I think the settings value itself is sufficient.